src/main/java/core/machines/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.machines;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
The stop message 'Bulldozer stopped working' could be more informative as well. Consider rephrasing it to 'Bulldozer has stopped working.' to make it clear that the Bulldozer has ceased operations.